packages/clerk-js/src/ui/contexts/CoreClerkContextWrapper.tsx

```diff
@@ -23,6 +23,7 @@ export function CoreClerkContextWrapper(props: CoreClerkContextWrapperProps): JS
     client: clerk.client,
     session: clerk.session,
     user: clerk.user,
+    organization: clerk.organization,
```

Comment:
This here just adds the state on the Wrapper. No effect currently but will be initially used in `useOrganization` with the addition of the ctx.